cmd/check_operator.go

```diff
@@ -122,5 +122,24 @@ var checkOperatorCmd = &cobra.Command{
 }
 
 func init() {
+	checkOperatorCmd.Flags().String("namespace", "", "The namespace to use when running OperatorSDK Scorecard. (env: PFLT_NAMESPACE)")
+	viper.BindPFlag("namespace", checkOperatorCmd.Flags().Lookup("namespace"))
+
+	checkOperatorCmd.Flags().String("serviceaccount", "", "The service account to use when running OperatorSDK Scorecard. (env: PFLT_SERVICEACCOUNT)")
+	viper.BindPFlag("serviceaccount", checkOperatorCmd.Flags().Lookup("serviceaccount"))
+
+	checkOperatorCmd.Flags().String("scorecard-image", "", "A uri that points to the scorecard image digest, used in disconnected environments.\n"+
+		"It should only be used in a disconnected environment. Use preflight runtime-assets on a connected \n"+
+		"workstation to generate the digest that needs to be mirrored. (env: PFLT_SCORECARD_IMAGE)")
+	viper.BindPFlag("scorecard_image", checkOperatorCmd.Flags().Lookup("scorecard-image"))
+
+	checkOperatorCmd.Flags().String("scorecard-wait-time", "", "A time value that will be passed to scorecard's --wait-time environment variable.\n"+
+		"(env: PFLT_SCORECARD_WAIT_TIME)")
+	viper.BindPFlag("scorecard-wait-time", checkOperatorCmd.Flags().Lookup("scorecard-wait-time"))
```

Comment:
The BindPFlag needs to be scorecard_wait_time, otherwise the envvar will be PFLT_SCORECARD-WAIT-TIME.

Reply:
Gotcha.